pages/manual/planet.md

```diff
@@ -143,6 +143,8 @@ By clicking on "Title", or "Rating", you are able to sort the course list accord
 
 * Action Menu
 
+By clicking the checkbox left of a course, you are able to use action menu.
```

Comment:
Action menu is related to each item so it is not required to click on checkbox. It shows from 3-dot icon

Reply:
That's weird. In my account, The 'manager actions' does not become active until I click the checkbox. Also, in the three dots does not have 'manager actions' option. Does my account has some issues? or did I make unclear in the sentence that I added?